extensions/wikia/Scribe/ScribeEventProducerController.class.php

```diff
@@ -59,65 +40,19 @@ static public function onSaveComplete( &$oPage, &$oUser, $text, $summary, $minor
 		return true;
 	}
 
-	static public function onSaveRevisionComplete( $oPage, $oRevision, $revision_id, $oUser, $allow = true ) {
+	static public function onArticleDeleteComplete( &$oPage, &$oUser, $reason, $page_id ) {
 		wfProfileIn( __METHOD__ );
 
-		# producer
-		if ( $allow ) {
-			$oScribeProducer = new ScribeEventProducer( 'edit' );
-			if ( is_object( $oScribeProducer ) ) {
-				/**
-				 * Ugly fix! TODO
-				 * See the description above.
-				 */
-				$oTitle = $oPage->getTitle();
-				if ( $oTitle->getNamespace() == NS_FILE ) {
-					self::$oPage = $oPage;
-					self::$oUser = $oUser;
-					self::$oRevision = $oRevision;
-				}
-
-				if ( $oScribeProducer->buildEditPackage( $oPage, $oUser, $oRevision ) ) {
-					$oScribeProducer->sendLog();
-				}
-			}
-		}
-
-		wfProfileOut( __METHOD__ );
-		return true;
-	}
-
-
-	static public function onDeleteComplete( &$oPage, &$oUser, $reason, $page_id ) {
-		wfProfileIn( __METHOD__ );
+		WikiaLogger::instance()->debug( "SUS-761::onArticleDeleteComplete", [
+			'page_id' => $page_id
+		] );
 
  		$oScribeProducer = new ScribeEventProducer( 'delete' );
 		if ( is_object( $oScribeProducer ) ) {
 			if ( $oScribeProducer->buildRemovePackage ( $oPage, $oUser, $page_id ) ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1066)